tests/qos/qos_sai_base.py

```diff
@@ -924,11 +927,44 @@ def updateDockerService(host, docker="", action="", service=""):  # noqa: F811
             duthost_upper.shell('sudo config feature state mux enabled')
             logger.info("Start mux container for dual ToR testbed")
 
-        enable_container_autorestart(
-            duthost, testcase="test_qos_sai", feature_list=feature_list)
         if 'dualtor' in tbinfo['topo']['name']:
             enable_container_autorestart(
-                duthost_upper, testcase="test_qos_sai", feature_list=feature_list)
+                duthost_upper,
+                testcase="test_qos_sai",
+                feature_list=feature_list)
+        else:
+            enable_container_autorestart(
+                duthost, testcase="test_qos_sai", feature_list=feature_list)
+
+    def updateDockerService(self, dut_asic, action=""):
+        """
+                 Helper function to update docker services
+
+                Args:
+                    dut_asic (AnsibleHost): Ansible host that is running docker
+                    docker (str): docker container name
+                    action (str): action to apply to service running within docker
+
+                Returns:
+                    None
+        """
+        services = [
+            {"docker": "lldp", "service": "lldp-syncd"},
+            {"docker": "lldp", "service": "lldpd"},
+            {"docker": "bgp", "service": "bgpd"},
+            {"docker": "bgp", "service": "bgpmon"},
+            {"docker": "radv", "service": "radvd"},
+            {"docker": "swss", "service": "arp_update"}
+        ]
+        for entry in services:
+            dut_asic.command(
+                "docker exec {docker} supervisorctl {action} {service}".format(
+                    docker=dut_asic.get_docker_name(entry['docker']),
```

Comment:
during test this changes, encounter below error:
```
AttributeError("'<class 'tests.common.devices.sonic.SonicHost'>' object has no attribute 'get_docker_name'",)
Traceback (most recent call last):
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/common/plugins/log_section_start/__init__.py", line 84, in _fixture_generator_decorator
    res = next(it)
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/qos/qos_sai_base.py", line 791, in fixture_stopServices
    upper_tor_host, lower_tor_host, toggle_all_simulator_ports)
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/qos/qos_sai_base.py", line 851, in stopServices
    self.updateDockerService(duthost, action="stop")
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/qos/qos_sai_base.py", line 922, in updateDockerService
    docker=dut_asic.get_docker_name(entry['docker']),
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/common/devices/multi_asic.py", line 112, in _run_on_asics
    return getattr(self.sonichost, self.multi_asic_attr)(*module_args, **complex_args)
  File "/home/xuchen3/env-a/sonic-mgmt-int/tests/common/devices/base.py", line 50, in __getattr__
    "'%s' object has no attribute '%s'" % (self.__class__, module_name)
AttributeError: '<class 'tests.common.devices.sonic.SonicHost'>' object has no attribute 'get_docker_name'
ERROR

... omitted ...

qos/qos_sai_base.py:791: in fixture_stopServices
    upper_tor_host, lower_tor_host, toggle_all_simulator_ports)
qos/qos_sai_base.py:851: in stopServices
    self.updateDockerService(duthost, action="stop")
qos/qos_sai_base.py:922: in updateDockerService
    docker=dut_asic.get_docker_name(entry['docker']),
common/devices/multi_asic.py:112: in _run_on_asics
    return getattr(self.sonichost, self.multi_asic_attr)(*module_args, **complex_args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <SonicHost str3-8101-02>, module_name = 'get_docker_name'

    def __getattr__(self, module_name):
        if self.host.has_module(module_name):
            self.module_name = module_name
            self.module = getattr(self.host, module_name)
    
            return self._run
        raise AttributeError(
>           "'%s' object has no attribute '%s'" % (self.__class__, module_name)
            )
E       AttributeError: '<class 'tests.common.devices.sonic.SonicHost'>' object has no attribute 'get_docker_name'
```

Reply:
@rraghav-cisco 
hit conflict when cherry pick changes to 202205 branch.
Could you please to create a PR to double commit to 202205?

Reply:
@XuChen-MSFT , which topology are you testing this on? When I ran on T1-64-lag, the dut_host object is SonicAsic.
(Pdb) dut_asic
<SonicAsic 0>
(Pdb) 
In your case, it appears as SonicHost. Can you pls clarify ?